src/pharmpy/tools/amd/run.py

```diff
@@ -59,9 +59,9 @@ def run_amd(
     results : ModelfitResults
         Reults of input if input is a model
     modeltype : str
-        Type of model to build. Either 'basic_pk' or 'tmdd'
+        Type of model to build. Either 'basic_pk', 'pk_iv', 'pk_oral', 'pk_ivoral'
```

Comment:
There is redundancy here. The route is specified twice

Reply:
I am not entirely sure i know what you are referring to?

Reply:
Model type 'pk_oral' for example includes the route so only `basic_pk` is needed for `modeltype`.

Reply:
Ah, I see! I thought we still wanted to have multiple options for `modeltype` but then I will update it to only have `basic_pk` and reduce the redundancy

Reply:
Yes, keep only `basic_pk` for now and we will add `tmdd`, `pkpd` etc later